README.org

```diff
@@ -1,23 +1,29 @@
 ** Zabbix Data Adapter
-*** Pre-requisites
-**** Zabbix
+
```

Comment:
Can you please rename this file to `README.md`. Not sure why we have a .org file.

Reply:
Not sure either. perhaps this README.org was created by someone long long time ago, in the galaxy far away...

Reply:
Changed the REAME.org to README.md